pkg/providers/amifamily/bootstrap/bottlerocketsettings.go

```diff
@@ -43,43 +43,54 @@ type BottlerocketSettings struct {
 
 // BottlerocketKubernetes is k8s specific configuration for bottlerocket api
```

Comment:
I know that you didn't write the comment on L38 but would you mind updating the comment to point to a valid filepath on BR? That will help us track the schema changes a lot easier. https://github.com/bottlerocket-os/bottlerocket/blob/develop/sources/models/src/lib.rs#L260